status-im/status-react
# feat: Add internal link preview for communities

fixes #17954

### Summary

Uses internal link preview component to show community link preview
Had to do a bit of coupling to make us of the mechanism that was already built by @ilmotta for link previews
![image](https://github.com/status-im/status-mobile/assets/33176106/1e75f91b-3c04-453d-ba98-2d1775c4cb81)
![image](https://github.com/status-im/status-mobile/assets/33176106/ec06e1c7-9367-4431-aa73-2cb4fdc2c900)

**QA NOTES:**

Please also check if this issue https://github.com/status-im/status-mobile/issues/17552 is reproducible in this PR. If not - please close that issue. 

status: ready
